lisa/tests/base.py

```diff
@@ -559,6 +559,7 @@ class RTATestBundle(TestBundle, metaclass=RTATestBundleMeta):
     """
 
     @requires_events('sched_switch')
+    @requires_one_event_of('rtapp_main', 'rtapp_task', 'rtapp_loop', 'rtapp_event', 'rtapp_stats')
```

Comment:
Is the intent to extract the same information from whatever event is present, or extract different information depending on what is available in the trace ? In the latter case, the new `may_use_events()` seems more appropriate.

Reply:
The second, will update.